src/main/java/com/minecolonies/commands/ListCitizens.java

```diff
@@ -0,0 +1,200 @@
+        final int citizenCount = citizens.size();
+
+        // check to see if we have to add one page to show the half page
+        int page = getIthArgument(args, 0, 1);
+        final int halfPage = (citizenCount % CITIZENS_ON_PAGE == 0) ? 0 : 1;
+        final int pageCount = ((citizenCount) / CITIZENS_ON_PAGE) + halfPage;
+
+        if (page < 1 || page > pageCount)
+        {
+            page = 1;
+        }
+
+        final int pageStartIndex = CITIZENS_ON_PAGE * (page - 1);
+        final int pageStopIndex = Math.min(CITIZENS_ON_PAGE * page, citizenCount);
+
+
+        List<CitizenData> citizensPage;
+
+        if (pageStartIndex < 0 || pageStartIndex >= citizenCount)
+        {
+            citizensPage = new ArrayList<>();
+        }
+        else
+        {
+            citizensPage = citizens.subList(pageStartIndex, pageStopIndex);
+        }
+
+        final ITextComponent headerLine = new TextComponentString(PAGE_TOP_LEFT + page + PAGE_TOP_MIDDLE + pageCount + PAGE_TOP_RIGHT);
+        sender.addChatMessage(headerLine);
+
+        for (final CitizenData citizen : citizensPage)
+        {
+            sender.addChatMessage(new TextComponentString(ID_TEXT + citizen.getId() + NAME_TEXT + citizen.getName()));
+
+            if (citizen.getCitizenEntity() != null)
+            {
+                final BlockPos position = citizen.getCitizenEntity().getPosition();
+                sender.addChatMessage(new TextComponentString(COORDINATES_TEXT + String.format(COORDINATES_XYZ, position.getX(), position.getY(), position.getZ())));
+            }
+        }
+        drawPageSwitcher(sender, colonyId, page, citizenCount, halfPage);
+    }
+
+    /**
+     * Get the ith argument (An Integer).
+     * @param i the argument from the list you want.
+     * @param args the list of arguments.
+     * @param def the default value.
+     * @return the argument.
+     */
+    private int getIthArgument(String[] args, int i, int def)
+    {
+        try
+        {
+            return Integer.parseInt(args[i]);
+        }
+        catch (NumberFormatException e)
+        {
+            return def;
+        }
+    }
+
+    /**
+     * Returns the colony of the owner or if not available colony 1.
+     * First tries to get the IColony and then the Colony from the ColonyManager.
+     * @param sender the sender of the command.
+     * @return the colonyId.
+     */
+    private int getColonyId( @NotNull final ICommandSender sender)
+    {
+        final IColony tempColony = ColonyManager.getIColonyByOwner(sender.getEntityWorld(), sender.getCommandSenderEntity().getUniqueID());
+        if(tempColony != null)
+        {
+            final Colony colony = ColonyManager.getColony(sender.getEntityWorld(), tempColony.getCenter());
+            if(colony != null)
+            {
+                return colony.getID();
+            }
+        }
+
+        return 1;
+    }
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)